.github/workflows/update-agent-changelog.yml

```diff
@@ -0,0 +1,71 @@
+name: Update the Agent changelog
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  update_agent_changelog:
+    name: Update the Agent Changelog
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        ref: master
```

Comment:
The default branch is the default

```suggestion
```

Reply:
That's what I understood from their doc at least https://github.com/actions/checkout?tab=readme-ov-file#usage, but maybe I misunderstood

Reply:
ah, misunderstood usage thanks